cmd/checkpoint/main.go

```diff
@@ -505,14 +545,85 @@ func (c *kubeletClient) localParentPods() map[string]*v1.Pod {
 	return podListToParentPods(podList)
 }
 
-// localRunningPods uses the /runningpods/ kubelet api to retrieve the local container runtime pod state
+// localRunningPods uses the CRI shim to retrieve the local container runtime pod state
```

Comment:
Can you add a brief comment explaining why you use the union of pods from both sandboxes and containers?

Reply:
added comment above the calls to obtain running sandboxes